src/pages-and-resources/proctoring/Settings.jsx

```diff
@@ -113,17 +120,21 @@ const ProctoringSettings = ({ intl, onClose }) => {
     }
 
     if (formValues.proctoringProvider === 'proctortrack') {
```

Comment:
Yeah this is a little weird since this does exist in both places. I think this make sense tho, trying to pass this back to platform probably creates more problems with no clear benefit.